### PR TITLE
Ray tune hparam with hydra

### PR DESCRIPTION
**Pull request by nilsleh**
This PR adds a python file to generate bash scripts that can then be run with the existing hydra setup for hyperparameter tuning with ray. It provides an outline of the moving pieces, but could be changed for different hparam strategies etc.

At the moment, the script tunes learning rate and batch size.

### CODE REVIEW

**Comment by ando-shah** [comment]
Would be great to have this feature!

**Comment by nilsleh** [comment]
@ando-shah I tested the script with dino and added some minimal instructions in the readme. let me know things that can be fixed, improved etc
